starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,37 @@
-// Names and Input
+let hacker1 = "Alfredo";
+console.log(`The driver's name is ${hacker1}`);
 
 
-//Conditionals
 
+let hacker2 = "Steve";
+console.log(`The navigator's name is ${hacker2}`);
 
-// Lorem ipsum generator
+if (hacker1.length > hacker2.length) {
+ console.log(`The Driver has the longest name, it has ${hacker1.length} characters`);
+} else if (hacker1.length < hacker2.length) {
+  console.log(`Yo, navigator got the longest name, it has ${hacker2.length} characters`);
+} else {
+  console.log(`wow, you both got equally long names, ${hacker1.length} characters!!`)
+}
+
+let hacker1UpperSpaced = hacker1.toUpperCase();
```

Comment:
Nice job condensing this step into just two lines! Same goes for the reversed string as well.